code/modules/power/apc/apc_main.dm

```diff
@@ -655,6 +663,29 @@
 		update_appearance(UPDATE_ICON)
 		update()
 
+/obj/machinery/power/apc/proc/install_cell_5k()
+	cell_type = /obj/item/stock_parts/cell/upgraded/plus
+	cell = new cell_type(src)
+
+/obj/machinery/power/apc/proc/install_cell_10k()
+	cell_type = /obj/item/stock_parts/cell/high
+	cell = new cell_type(src)
+
+/obj/machinery/power/apc/proc/unlock()
+	locked = 0
+
+/obj/machinery/power/apc/proc/give_syndicate_access()
+	req_access = list(ACCESS_SYNDICATE)
+
+/obj/machinery/power/apc/proc/give_away_general_access()
+	req_access = list(ACCESS_AWAY_GENERAL)
+
+/obj/machinery/power/apc/proc/set_no_charge()
+	cell.charge = 0
+
+/obj/machinery/power/apc/proc/set_full_charge()
+	cell.charge = 100
```

Comment:
dmdoc all of these too for statistical reasons. doesn't have to be anything fancy